java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -255,6 +265,14 @@ private boolean learnedConstraintForDomains(LearnedConstraint lc) {
         })
         .forEach(flowBuilder::add);
 
+      return flowBuilder.build();
+    }
+
+    private List<JavaFileScannerContext.Location> flowFromYields(ExplodedGraph.Edge edge) {
+      ExplodedGraph.Node parent = edge.parent;
+      List<Integer> argumentIndices = correspondingArgumentIndices(symbolicValues, parent);
+      SymbolicValue returnSV = edge.child.programState.peekValue();
+      ImmutableList.Builder<JavaFileScannerContext.Location> flowBuilder = ImmutableList.builder();
       MethodYield selectedMethodYield = edge.child.selectedMethodYield(parent);
       if (selectedMethodYield != null) {
```

Comment:
See previous comment on the file but this test should be nearly the first thing in the method and that will remove a useless and you'l compute argument indices only if the yield is not null.